packages/client-sdk-nodejs-compression-zstd/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@gomomento/sdk-nodejs-compression-zstd",
+  "version": "0.0.1",
+  "description": "Compression extensions for Momento node.js SDK",
```

Comment:
Include details on that this is for zstd?

Reply:
done in https://github.com/momentohq/client-sdk-javascript/pull/1264/commits/93d0c1a97da7fde7e4a9a225ea7b2a7caab890d4